src/Workspaces/Core/Portable/FindSymbols/FindReferences/Finders/NamespaceSymbolReferenceFinder.cs

```diff
@@ -47,12 +47,12 @@ private static string GetNamespaceIdentifierName(INamespaceSymbol symbol)
             var identifierName = GetNamespaceIdentifierName(symbol);
             var syntaxFactsService = document.GetLanguageService<ISyntaxFactsService>();
 
-            var nonAliasReferences = await FindReferencesInTokensAsync(symbol,
+            var nonAliasReferences = FindReferencesInTokens(symbol,
```

Comment:
consider extracting out hte Async->Sync change so it can go in separately.  that wuold help make this PR smaller and easier to review.